dobackup.py

```diff
@@ -35,17 +35,24 @@ def UIDFromFilename(fname):
 
 
 def get_credentials():
-    user = raw_input("Gmail address: ")
-    pwd = getpass.getpass("Gmail password: ")
+    user = os.environ.get("DOBACKUP_GMAIL_APP_USER", None) or raw_input("Gmail address: ")
+    pwd = os.environ.get("DOBACKUP_GMAIL_APP_PWD", None) or getpass.getpass("Gmail password: ")
     return user, pwd
 
+def gmail_folder():
+    return os.environ.get("DOBACKUP_GMAIL_LABEL", GMAIL_FOLDER_NAME) 
+
+def save_folder_path():
+    folder = os.environ.get("DOBACKUP_SAVE_FOLDER", "") # i.e. "./"
+    os.makedirs(folder, exist_ok=True)
```

Comment:
I think this will fail if folder is "" (at least for some os/version combinations). Can you change it to only run if folder is not "".